src/Files.App/Helpers/UI/UIFilesystemHelpers.cs

```diff
@@ -87,7 +87,7 @@ public static async Task CreateFileFromDialogResultTypeAsync(AddItemDialogItemTy
 			string? userInput = null;
 			if (itemType != AddItemDialogItemType.File || itemInfo?.Command is null)
 			{
-				DynamicDialog dialog = DynamicDialogFactory.GetFor_RenameDialog();
+				DynamicDialog dialog = DynamicDialogFactory.GetFor_CreateItemDialog(itemInfo?.Name ?? itemType.ToString().ToLower());
```

Comment:
```suggestion
				DynamicDialog dialog = DynamicDialogFactory.GetFor_CreateItemDialog(itemType.ToString().GetLocalizedResource());
```
This way should work fine. `itemInfo` is not very useful because it is often null.